.github/workflows/workflow.yml

```
@@ -104,8 +110,13 @@ jobs:
       - name: Setup | Checkout
         uses: actions/checkout@v2
 
-      - name: Setup | Cache Cargo
-        uses: actions/cache@v2
+      - name: Install libdbus (ubuntu)
+        if: matrix.os == 'ubuntu-latest'
+        run: sudo apt-get install dbus libdbus-1-3 libdbus-glib-1-dev
+
+      # Cache files between builds
+      - name: Cache cargo registry
+        uses: actions/cache@v1
```

We only need `libdbus-1-dev` because `dbus` and `libdbus-1-3` is preinstalled and `libdbus-glib-1-dev` is not needed.

```suggestion
      - name: Setup | libdbus (ubuntu)
        if: matrix.os == 'ubuntu-latest'
        run: sudo apt-get install libdbus-1-dev

      # Cache files between builds
      - name: Setup | Cache Cargo
        uses: actions/cache@v2
```